app/views/users/show.html.erb

```diff
@@ -1 +1,3 @@
-<p>Users#show</p>
\ No newline at end of file
+<%= render partial:"user", locals: {user: @user} %>
```

Comment:
Please use the `recent_posts` scope from the `User` model in `User's show page` to render the user's 3 most recent posts and there should be a `See all Posts` button as can be seen in this [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_user_page.png) which takes the user to the users' index page on click.

Reply:
I switched to using the recent_posts scopes instead of listing all the posts, thanks for the tip 👍 💯